.env

```diff
@@ -14,6 +14,7 @@ LOGO_WHITE_URL=''
 FAVICON_URL=''
 MARKETING_SITE_BASE_URL=''
 ORDER_HISTORY_URL=''
+POST_MARK_AS_READ_DELAY=null
```

Comment:
why passing null here?

Reply:
A blank string didn't seem to match the type here, which is a number.

Reply:
why not define the default value here?

Reply:
Will do.